pm2.json

```diff
@@ -1,4 +1,15 @@
 {
   "apps": [
+    {
+      "name": "@ove/ove-service-layout",
+      "cwd": "./packages/ove-service-layout",
+      "kill_timeout": 3000,
+      "restart_delay": 3000,
+      "script": "npm",
+      "args": "run start",
+      "env": {
+          "PORT": "8186"
```

Comment:
port range should be agreed. Currently ove-services have a port range of 8181 onwards. This is the only service since assetmanager is a separate OVE component. Therefore, this service should ideally have port 8181. Anyway this is not a blocking issue for this PR.

Reply:
We can change the port when the install/deployment/dev ops project will be created.